app/http.php

```diff
@@ -43,10 +43,40 @@
 $payloadSize = 12 * (1024 * 1024); // 12MB - adding slight buffer for headers and other data that might be sent with the payload - update later with valid testing
 $totalWorkers = intval(System::getEnv('_APP_CPU_NUM', swoole_cpu_num())) * intval(System::getEnv('_APP_WORKER_PER_CORE', 6));
 
+/**
+ * Assigns HTTP requests to worker threads by analyzing its payload/content.
+ *
+ * Routes requests as 'safe' or 'risky' based on specific content patterns (like POST actions or certain domains)
+ * to optimize load distribution between the workers. Utilizes `$safeThreadsPercent` to manage risk by assigning
+ * riskier tasks to a dedicated worker subset. Prefers idle workers, with fallback to random selection if necessary.
+ * doc: https://openswoole.com/docs/modules/swoole-server/configuration#dispatch_func
+ *
+ */
+$swooleDispatchFunction = new class ($domains) extends \Utopia\Swoole\Dispatch\RiskyRequest {
+    public function __construct(private readonly Table $domains)
+    {
+        parent::__construct(1024 * 1024, intval(System::getEnv('_APP_RISKY_WORKERS_PERCENT', 80)) / 100);
+    }
+
+    protected function isRisky(string $request, string $domain): bool
+    {
+        $risky = false;
+        if (str_starts_with($request, 'POST') && str_contains($request, '/executions')) {
+            $risky = true;
+        } elseif (str_ends_with($domain, System::getEnv('_APP_DOMAIN_FUNCTIONS'))) {
+            $risky = true;
+        } elseif ($this->domains->get(md5($domain), 'value') === 1) {
+            // executions request coming from custom domain
+            $risky = true;
+        }
+        return $risky;
```

Comment:
```suggestion
        if (str_starts_with($request, 'POST') && str_contains($request, '/executions')) {
            return true;
        } elseif (str_ends_with($domain, System::getEnv('_APP_DOMAIN_FUNCTIONS'))) {
            return true;
        } elseif ($this->domains->get(md5($domain), 'value') === 1) {
            // executions request coming from custom domain
            return true;
        }
        return false;
```

we can probably skip the var 👍🏻